components/shutter_static_webapp/static_webapp.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  shutter_apps = [for k, v in var.frontends : v if lookup(v, "shutter_app", false)]
+  shutter_apps = [for k, v in var.frontends : v if lookup(v, "shutter_app", true)]
```

Comment:
Just so I'm sure, this will check for a key named shutter_app in the tfvars? If so, why are [these values](https://github.com/hmcts/azure-platform-terraform/pull/1654/files#diff-f49daa1c100e9f0826b25e3993197866e8dc25c67113ecd2db1eba6d98da31b1R642) required?

Reply:
changing the default value to true. Shutter infra is deployed by default. We need to add **shutter_app = false** if don't it. This is opposite to the logic we currently have